apps/explorer/lib/explorer/celo/telemetry/instrumentation/blockchain.ex

```diff
@@ -59,6 +59,24 @@ defmodule Explorer.Celo.Telemetry.Instrumentation.Blockchain do
         event_name: [:indexer, :tokens, :average_gas],
         measurement: :value,
         description: "Average gas used over last n blocks"
+      ),
+
+      # NFT
+      last_value(
```

Comment:
This is understandably confusing but the rest of the values in the file are intended to represent the state of the blockchain - i.e. not so much "what is the current indexed status of these items" as "what is the current  state of the blockchain for value x". 

Could you move these metrics to the indexer at `apps/indexer/lib/indexer/celo/telemetry/instrumentation/indexer.ex` ?

Things like average gas used and most recent block number should be able to be called from any pod, whilst indexer ingested stats are only from the indexer.

Again, it is pretty confusing because all the values in this file also have `:indexer` in the event name, but this is for backwards compatibility as the comment above says 

https://github.com/celo-org/blockscout/pull/880/files#diff-e503c3420804e4ab858b3bf956685d2b8606bf1cb2544fcac91d62d7a4af0cc8L8